README.md

```diff
@@ -40,12 +42,13 @@ The middleware function takes an object as argument
 import validate, { Joi } from ('koa-router-joi-validation');
 .....
     validate({
-      query: // Joi schema object
-      body: // Joi schema object
-      params: // Joi schema object
-      headers: // Joi schema object
-      200: // Joi schema object
-      503: // Joi schema object
+      query: // Joi schema definition
+      body: // Joi schema definition
+      params: // Joi schema definition
+      headers: // Joi schema definition
+      schema: // Compiled Joi schema object
```

Comment:
what is the difference between  `Compiled Joi schema object` and `Joi schema`?

Reply:
if it's the same we should keep the terms

Reply:
we have simple JS that is transformed to joi.object. Compiled is already Joi object and don't need to call Joi.object function

Reply:
they call it compilation of JS object to Joi object on their site

Reply:
So what I want to say there is that if you validate query, params and others separately with old version you need to pass JS object. But for schema need to pass Joi object